<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KaliscopeWin</class>
 <widget class="QMainWindow" name="KaliscopeWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaliscope film scanner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/kaliscopeResources.qrc">
    <normaloff>:/kaliscope/icons/app/kaliscope.png</normaloff>:/kaliscope/icons/app/kaliscope.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="layoutVideo">
      <item>
       <widget class="QOpenGLWidget" name="openglView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblCurrentTrack">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSlider" name="sliderPosition">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblTrackLength">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QToolButton" name="btnSettings">
          <property name="text">
           <string>S</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../libraries/mvp-player/libraries/mvp-player-qtgui/src/mvp-player-qtgui/resources/mvpplayer.qrc">
            <normaloff>:/mvpplayer/toolbar/applications-settings.png</normaloff>:/mvpplayer/toolbar/applications-settings.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layoutButtonsBar">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>btnSettings</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../libraries/mvp-player/libraries/mvp-player-qtgui/src/mvp-player-qtgui/resources/mvpplayer.qrc"/>
  <include location="../resources/kaliscopeResources.qrc"/>
 </resources>
 <connections/>
</ui>
